### PR TITLE
Updated Batch Track 2 Management plane SDK for 2022-10-01 API version

### PR DESCRIPTION
**Pull request by NickKouds**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/batch/Azure.ResourceManager.Batch/Azure.ResourceManager.Batch.sln [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.32929.385
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{55ED41CE-5314-4E12-BDF8-024A5DC5381B}") = "Azure.ResourceManager.Batch", "src\Azure.ResourceManager.Batch.csproj", "{CD6780C7-6290-42AB-B5CD-F8CFAF1248DE}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.Batch", "src\Azure.ResourceManager.Batch.csproj", "{CD6780C7-6290-42AB-B5CD-F8CFAF1248DE}"
 EndProject
-Project("{55ED41CE-5314-4E12-BDF8-024A5DC5381B}") = "Azure.ResourceManager.Batch.Tests", "tests\Azure.ResourceManager.Batch.Tests.csproj", "{97228A51-6A22-4C1C-9534-037AD1A33016}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.Batch.Tests", "tests\Azure.ResourceManager.Batch.Tests.csproj", "{97228A51-6A22-4C1C-9534-037AD1A33016}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{7503DF06-79BD-462A-BA9E-D421EBC0AC12}"
```
Comment (dpwatrous) [inline sdk/batch/Azure.ResourceManager.Batch/Azure.ResourceManager.Batch.sln:10]: Can we remove this?

File: sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 2.0.0 (2022-10-01)
```
Comment (dpwatrous) [inline sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md:3]: This should be the release date, not the API version

**Review by dpwatrous — CHANGES_REQUESTED** [review]

**Review by dpwatrous — APPROVED** [review]
Looks good to me

File: sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md [hunk 3]
```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 2.0.0 (2022-11-07)
```
Comment (fengzhou-msft) [inline sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md:3]: Please don't bump major version for track 2.

File: sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md [hunk 4]
```diff
@@ -1,15 +1,25 @@
...
+  - This property determines how a pool communicates with the Batch service.
+  - Possible values: Default, Classic, Simplified.
+- Added properties `CurrentNodeCommunicationMode` and `TargetNodeCommunicationMode` of type `NodeCommunicationMode` to `BatchAccountPoolData`.
 
 ### Breaking Changes
 
-### Bugs Fixed
+- Modified casing of Username property of the `BatchCifsMountConfiguration` class to `UserName`.
+- Modified casing of DynamicVNetAssignmentScope property of the `BatchNetworkConfiguration` class to `DynamicVnetAssignmentScope`.
+- Modified casing of ActionRequired property of the `BatchPrivateLinkServiceConnectionState` class to `ActionsRequired`.
```
Comment (fengzhou-msft) [inline sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md:16]: Breaking changes are not allowed. Please use configuration or customized code to make them backward compatible.

File: sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md (hunk 3 above)
Comment (NickKouds) [reply sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md]: @fengzhou-msft - Changed version number by bumping minor version number.

File: sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md (hunk 4 above)
Comment (NickKouds) [reply sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md]: @fengzhou-msft Reverted back breaking changes.

File: sdk/batch/Azure.ResourceManager.Batch/src/Azure.ResourceManager.Batch.csproj [hunk 5]
```diff
@@ -1,8 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.1.0-beta.1</Version>
-    <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.0.0</ApiCompatVersion>
+    <Version>1.1.0</Version>
```
Comment (fengzhou-msft) [inline sdk/batch/Azure.ResourceManager.Batch/src/Azure.ResourceManager.Batch.csproj:3]: Please add back `<ApiCompatVersion>1.0.0</ApiCompatVersion>`.

**Comment by fengzhou-msft** [comment]
There are some CredScan failures. Please [add them to suppressions](https://github.com/Azure/azure-sdk-for-net/blob/main/doc/dev/CredscanProcess.md#false-positives) if they're false alarm or [sanitize them with JsonPathSanitizers](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md#sanitizing) if they are real security leaks.

File: sdk/batch/Azure.ResourceManager.Batch/src/Azure.ResourceManager.Batch.csproj (hunk 5 above)
Comment (fengzhou-msft) [reply sdk/batch/Azure.ResourceManager.Batch/src/Azure.ResourceManager.Batch.csproj:3]: Yes, but we need it in this PR to run API compatibility check for `1.1.0` with the released `1.0.0`.

Comment (NickKouds) [reply sdk/batch/Azure.ResourceManager.Batch/src/Azure.ResourceManager.Batch.csproj]: Added it back.

**Pull request merged**